service/health/types.go

```diff
@@ -117,7 +119,19 @@ func NewClusterStatus(cluster v1alpha1.StatusCluster, nodes []v1.Node) ClusterSt
 	}
 }
 
-func NewNodeStatus(node v1.Node) NodeStatus {
+func NewNodeStatus(node v1.Node, pods []v1.Pod) NodeStatus {
+	limits := NodeStatusComputeResources{}
+	requests := NodeStatusComputeResources{}
+	for _, pod := range pods {
+		if pod.Spec.NodeName == node.Name {
+			for _, container := range pod.Spec.Containers {
+				limits.CPU += container.Resources.Limits.Cpu().MilliValue()
```

Comment:
What other possible values are there? Are we sure we want to stick with Milli?

Reply:
The type is `resource.Quantity`. It can be scaled arbitrarily using SI suffixes like 1M, 1m, etc ([see source here](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go)). According to [the Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu), CPU can be no smaller than 1m (0.001 of one core) so I thought returning millis makes sure that we don't have to deal with decimals without losing precision. Also it's how kubectl shows CPU requests/limits.